src/gromacs/nblib/gmxcalculator.h

```diff
@@ -106,6 +106,9 @@ class GmxForceCalculator
     matrix box_;
 };
 
+//! Puts particles on a grid based on bounds specified by the box
+void setParticlesOnGrid(SimulationState simulationState, std::unique_ptr<nonbonded_verlet_t>& nbv, std::vector<int>& particleInfoAllVdw_);
```

Comment:
The argument list should be the parameters that are actually used and not the whole simState object so that the data flow is clear.

Reply:
Done.